src/samples/coap/lwm2m-client.c

```diff
@@ -293,23 +342,181 @@ read_security_server_obj(void *instance_data, void *user_data,
 }
 
 static int
+write_security_res(void *instance_data, void *user_data,
+    struct sol_lwm2m_client *client,
+    uint16_t instance_id, uint16_t res_id, const struct sol_lwm2m_resource *res)
+{
+    struct security_obj_instance_ctx *instance_ctx = instance_data;
+    int r;
+
+    // res_id?
```

Comment:
It's just that it seems I could actually use `res->id` where `res_id` appears, and thus this parameter seems to be unnecessary... is that right?